src/System.Diagnostics.Process/tests/Interop.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace System.Diagnostics.Tests
 {
-    internal class Interop
```

Comment:
Well this one is actually shared, not Windows specific, which is why I didn't add the suffix.